server/src/main/java/org/opensearch/common/blobstore/stream/write/UploadResponse.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.blobstore.stream.write;
+
+/**
+ * Response object for uploads using <code>BlobContainer#writeBlobByStreams</code>
+ */
+public class UploadResponse {
```

Comment:
Wouldn't the ` writeBlobByStreams` throw IOCException if there are issues with the upload. Wouldn't the response would always be true or an exception thrown otherwise. We could have `CompletableFuture<Void>` as well, right? Is there a possibility that this class is not required for now?

Reply:
I think we can consider removing it. @raghuvanshraj